### PR TITLE
Add loading data animation, swipe gesture and fix move-up animation

### PR DESCRIPTION
**Pull request by trangttdang**

### CODE REVIEW

File: Farmacy/Models/Conversation.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+//
+//  Created by Trang Dang on 8/1/22.
+//
+
+#import <Parse/Parse.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Conversation : PFObject <PFSubclassing>
+@property (nonatomic, strong) PFUser *userOne;
```
Comment (xmdou) [inline Farmacy/Models/Conversation.h]: Are userOne and userTwo for sender and receiver? If so, you can name them them sender and receiver. Thanks.

Comment (trangttdang) [reply Farmacy/Models/Conversation.h]: Hi May! Conversation is between two users, both of them can be sender or receiver. That's why I cannot define who is sender or receiver in Conversation model. But by fetching all messages (Message model) that has Conversation ID along with message sender, I can know who sent this message (for example userOne), the another user of conversation (userTwo) will be receiver.

File: Farmacy/Delegate/AppDelegate.m [hunk 2]
```diff
@@ -31,7 +31,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
         
         configuration.applicationId = [dict objectForKey:@"parse_app_id"];
         configuration.clientKey = [dict objectForKey:@"parse_client_key"];
-        configuration.server = @"https://parseapi.back4app.com";
+        configuration.server = @"wss://farmacy.b4a.io";
```
Comment (jianghaolu) [inline Farmacy/Delegate/AppDelegate.m]: This was done in another PR right? You will need to resolve the conflicts before merging

File: Farmacy/Models/Conversation.m [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+//  Farmacy
+//
+//  Created by Trang Dang on 8/1/22.
+//
+
+#import "Conversation.h"
+
+@implementation Conversation
+@dynamic userOne;
+@dynamic userTwo;
```
Comment (jianghaolu) [inline Farmacy/Models/Conversation.m]: I'm not familiar with the syntax of `@dynamic` in objc, but I thought these were already defined as properties in Conversation.h?

File: Farmacy/Models/Message.m [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+//
+//  Created by Trang Dang on 8/1/22.
+//
+
+#import "Message.h"
+
+@implementation Message
+@dynamic text;
+@dynamic conversation;
+@dynamic sender;
```
Comment (jianghaolu) [inline Farmacy/Models/Message.m]: Same here

File: Farmacy/Views/ChatCell.h [hunk 5]
```diff
@@ -1,16 +0,0 @@
-//
```
Comment (jianghaolu) [inline Farmacy/Views/ChatCell.h]: Do you want to delete `ChatCell.m` too?

**Review by jianghaolu — APPROVED** [review]
You will have some merge conflicts to resolve
